plugins/gtag.ts

```diff
@@ -0,0 +1,12 @@
+export default defineNuxtPlugin(() => {
+  const { gtag } = useGtag()
+  const googleAnalyticsTrackingId = useRuntimeConfig().public.googleAnalyticsTrackingId as string
```

Comment:
![image](https://github.com/user-attachments/assets/c137202f-c197-4871-bb5f-afb2f09afce8)

```suggestion
  const googleAnalyticsTrackingId = useRuntimeConfig().public.googleAnalyticsTrackingId
```